src/app/forms.py

```diff
@@ -55,4 +60,14 @@ class OrgInfoForm(forms.ModelForm):
     
     class Meta:
         model = UserID
-        fields = ('organisation',)
\ No newline at end of file
+        fields = ('organisation',)
+
+class LicenseRequestForm(forms.Form):
```

Comment:
You can use the `class Meta:` with model `LicenseRequest`. See other forms for examples. You would not need to add forms.Charfield for every variable. Also `class Meta:` provides some inbuilt checking - like if fields are empty/unique/etc. So it would be best to use the Meta class for forms.

Reply:
Ok, I will  look how this works. I actually do not know how this could be used, then I will see other examples.

Reply:
Just see the above examples of UserRegisterForm, UserProfileForm. It's right above the class you made.